core-java-modules/core-java-io-5/src/test/java/com/baeldung/charstreams/CharStreamsUnitTest.java

```diff
@@ -0,0 +1,116 @@
+package com.baeldung.charstreams;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.*;
+import java.nio.file.*;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.jupiter.api.*;
+
+public class CharStreamsUnitTest {
+
+    private static final DailyTodo TODO = new DailyTodo("Code", LocalDateTime.now(), true);
+    private static final DailyTodo TODO_1 = new DailyTodo("Sleep", LocalDateTime.now(), false);
+    private static final DailyTodo TODO_2 = new DailyTodo("Read", LocalDateTime.now(), false);

```

Comment:
I also just realised that there is no reason for us to have multiple objects here, since you only ever use one object per test. So, you can delete TODO_1 and TODO_2 and just use TODO. The reason for this change is that it can confuse the reader as they search to understand why there are 3 different variables used in the tests.